### PR TITLE
Jet v1.4.4 update, js migrations and doc updates

### LINKED ISSUE #1: Interfaces page provides URLs that don't work
https://embassy-myhexid.local/#/services/lightning-jet/interfaces shows :

># Network Interface
>Specifies the interface to listen on for HTTP connections.
>Tor Address
>http://mypersonalembassycrazylongonionaddress.onion
>LAN Address
>https://mypersonalembassycrazylongonionaddress.local

Selecting either of these addresses results in a 502 Bad Gateway...

Under instructions it does say "Lightning Jet is command-line only service"

This page should repeat that information (no web UI available).

### LINKED ISSUE #2: Update Instructions to make it clear that when Lightning Jet is started, it immediately starts to rebalance.
After installation, when you start lightning jet it immediately tries to rebalance - and can spend your sats on this... 

Please add a note to the top of instructions. Something like:

***NOTE:*** When this service is started it will automatically start rebalancing attempts.  Please review https://github.com/itsneski/lightning-jet for more details on how this service works...

### LINKED ISSUE #3: [chore]: upstream update of Lightning Jet to v1.4.4

### LINKED ISSUE #4: Add log file location to the instructions[feat]:
### Prerequisites

- [X] I have searched for existing issues that already suggest this feature.

### Describe the Feature Request

Include the log location / how to see logs on the instruction page.

Lightning Jet has 4 main services.  You can see their status when SSH'd in to Embassy  and do a "jet status". This also shows their location.

>│ (index) │   service    │  status   │          log           │
>├─────────┼──────────────┼───────────┼────────────────────────┤
>│    0    │ 'rebalancer' │ 'running' │ '/tmp/rebalancer.log'  │
>│    1    │   'logger'       │ 'running' │ '/tmp/htlc-logger.log' │
>│    2    │   'daddy'       │ 'running' │  '/tmp/launcher.log'   │
>│    3    │  'telegram'   │ 'running' │  '/tmp/telegram.log'

Finding the files -  was tricky, so providing instructions might help.

### Describe the Use Case

After logging in to docker running bash, the log file directory is actually up a level so may not be easy to find. 
>start9@embassy-myhexid:/tmp $ sudo docker exec -it lightning-jet.embassy /bin/bash
>bash-5.1# pwd
>/app
>bash-5.1# ls -l
>total 584
>-rw-r--r--    1 root     root          1093 Jun  3 01:11 LICENSE
>-rw-r--r--    1 root     root         14532 Jun  3 01:11 README.md
>drwxr-xr-x    1 root     root          4096 Jul 26 18:06 api

Trying either
>start9@embassy-myhexid:/tmp $ sudo docker exec -it lightning-jet.embassy ls -l /tmp
>total 668
>-rw-r--r--    1 root     root          7282 Jul 26 20:11 htlc-logger.log
>-rw-r--r--    1 root     root         11597 Jul 26 20:11 launcher.log
>-rw-r--r--    1 root     root        240953 Jul 26 20:16 rebalancer.log
>-rw-r--r--    1 root     root         72071 Jul 26 20:11 telegram.log
>-rw-------    1 root     root        265969 Jun  2 20:25 tmp2gcp4r3kcacert.pem

or 

>start9@embassy-743d6982:/tmp $ sudo docker exec -it lightning-jet.embassy /bin/bash
>bash-5.1# cd ../tmp
>bash-5.1# ls -l
>total 668
>-rw-r--r--    1 root     root          7282 Jul 26 20:11 htlc-logger.log
>-rw-r--r--    1 root     root         11597 Jul 26 20:11 launcher.log
>-rw-r--r--    1 root     root        240953 Jul 26 20:16 rebalancer.log
>-rw-r--r--    1 root     root         72071 Jul 26 20:11 telegram.log
>-rw-------    1 root     root        265969 Jun  2 20:25 tmp2gcp4r3kcacert.pem

commands will get you to the log directory.

### Describe Preferred Solution

Update the instructions to include this information about how to find the logs

### Describe Alternatives

Make the logs display on the logs page (not sure how you would display multiple logs there...

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by islandbitcoin**
- [x] #3 
- [x] #4 
- [x] #2 
- [x] #1

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -69,24 +46,10 @@ volumes:
...
-        80: "80"
-    lan-config:
-      443:
-        ssl: true
-        internal: 80
-    ui: false
-    protocols:
-      - tcp
-      - http
+interfaces: {}
```
Comment (chrisguida) [inline manifest.yaml:49]: Why did you delete the interface?

File: manifest.yaml [hunk 2]
```diff
@@ -119,4 +82,13 @@ backup:
...
+      main: "/root/data"
+migrations:
+  from:
+    "*":
+      type: script
+      args: ["from"]
+  to:
+    "*":
+      type: script
+      args: ["to"]
```
Comment (chrisguida) [inline manifest.yaml]:
```suggestion
      args: ["to"]

```

File: scripts/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+*.js
```
Comment (chrisguida) [inline scripts/.gitignore]:
```suggestion
*.js

```

Comment (chrisguida) [reply scripts/.gitignore]: Are there lots of `.js` files being ignored? isn't it just `scripts/embassy.js`? if so you can just put this in the parent dir `.gitignore`

File: scripts/deps.ts [hunk 4]
```diff
@@ -0,0 +1,2 @@
+export * from "https://deno.land/x/embassyd_sdk@v0.3.1.1.1/mod.ts";
+export * from "https://deno.land/x/embassyd_sdk@v0.3.1.0.12/emvar-lite/mod.ts";
```
Comment (chrisguida) [inline scripts/deps.ts]: What's this for? also why is it misspelled? Should be `emver`, not `emvar`

File: scripts/embassy.ts [hunk 5]
```diff
@@ -0,0 +1,4 @@
+export { setConfig } from "./services/setConfig.ts";
+export { properties } from "./services/properties.ts";
+export { getConfig } from "./services/getConfig.ts";
+export { migration } from "./services/migrations.ts";
```
Comment (chrisguida) [inline scripts/embassy.ts]:
```suggestion
export { migration } from "./services/migrations.ts";

```

File: scripts/services/migrations.ts [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+          (config) => {
+            return config;
+          },
+          true,
+          { version: "1.4.4", type: "down" },
+        ),
+      },
+    },
+    "1.4.4",
+  );
```
Comment (chrisguida) [inline scripts/services/migrations.ts]:
```suggestion
  );

```

Also, this migration doesn't appear to do anything?

Also, this is a self-migration?

File: scripts/services/properties.ts [hunk 7]
```diff
@@ -0,0 +1,3 @@
+import { compat, types as T } from "../deps.ts";
+
+export const properties: T.ExpectedExports.properties = compat.properties;
```
Comment (chrisguida) [inline scripts/services/properties.ts]:
```suggestion
export const properties: T.ExpectedExports.properties = compat.properties;

```

File: scripts/services/setConfig.ts [hunk 8]
```diff
@@ -0,0 +1,3 @@
+import { compat } from "../deps.ts";
+
+export const setConfig = compat.setConfig;
```
Comment (chrisguida) [inline scripts/services/setConfig.ts]:
```suggestion
export const setConfig = compat.setConfig;

```

**Review by chrisguida — CHANGES_REQUESTED** [review]
See comments

File: manifest.yaml (hunk 1 above)
Comment (islandbitcoin) [reply manifest.yaml:49]: Jesse opened an issue for it. Jet doesn't use the interface at all, its strictly a local CLI tool

File: scripts/deps.ts (hunk 4 above)
Comment (islandbitcoin) [reply scripts/deps.ts]: I don't think we need the 2nd line anymore. I'll remove it

File: scripts/services/migrations.ts (hunk 6 above)
Comment (islandbitcoin) [reply scripts/services/migrations.ts]: self-migration, made it easier to test. also doesn't do anything from other versions. any issue with that?

Comment (chrisguida) [reply scripts/services/migrations.ts]: I thought migrations that aren't defined are now `configured: true`? @dr-bonez ?

File: manifest.yaml (hunk 1 above)
Comment (chrisguida) [reply manifest.yaml:49]: What was the problem, and how does this fix?

Comment (islandbitcoin) [reply manifest.yaml:49]: https://github.com/Start9Labs/lightning-jet-wrapper/issues/1

Comment (chrisguida) [reply manifest.yaml:49]: Yes, I already looked at it. The reason I asked the question is because it doesn't say what the problem was or what the fix is.

File: manifest.yaml [hunk 9]
```diff
@@ -1,7 +1,9 @@
 id: lightning-jet 
 title: Lightning Jet
-version: 1.4.3
-release-notes: Initial Release to EmbassyOS v0.3.0.3
+version: 1.4.4
+release-notes: |
+  * Update to Jet v1.4.4-embassy
```
Comment (chrisguida) [inline manifest.yaml]: What is "Jet v1.4.4-embassy"?

Comment (islandbitcoin) [reply manifest.yaml]: this is the latest release tag from the upstream repo. 
https://github.com/itsneski/lightning-jet/releases/tag/v1.4.4-embassy


it seems many of your questions during PR reviews can be answered by looking at the upstream repo. is there additional detail I should be putting in this PR to pro-actively give you this info?

Comment (chrisguida) [reply manifest.yaml]: Yes, please pull in/link any relevant info from upstream. 

First, I don't want to be having to search through the upstream repo for relevant info. 
Second, I've never seen an upstream repo release a version tagged with `-embassy`, so thanks for linking that; I definitely wouldn't have thought to search around the upstream repo for that.

Comment (chrisguida) [reply manifest.yaml]: Probably just include that link in the release notes, so people aren't confused

Comment (chrisguida) [inline manifest.yaml]:
```suggestion
  * Update to [Jet v1.4.4-embassy](https://github.com/itsneski/lightning-jet/releases/tag/v1.4.4-embassy)
```

**Review by chrisguida — CHANGES_REQUESTED** [review]
see comment
